DECISION-MAKING.md

```diff
@@ -0,0 +1,99 @@
+# Decision-making rules
+
+## Introduction
+
+### Maintainers Group
+
+| Name                                                | Time commitment | Role / Scope |
+| --------------------------------------------------- | --------------- | ------------ |
+| Rémi Gau ([@Remi-Gau](https://github.com/Remi-Gau)) | 5h/week         | Admin        |
+| Guillaume Flandin ([ ?? ](https://github.com/ ??))  | ?? h/week       | Admin        |
+| Robert Oostenvold ([ ?? ](https://github.com/ ??))  | ?? h/week       | Admin        |
+
+Maintainers may declare a limited scope of responsibility. Such a scope can
+range from maintaining a modality supported in the specification to nurturing a
+welcoming community. One or more scopes can be chosen by the maintainer and
+agreed upon by the Maintainers Group. A maintainer is primarily responsible for
+issues within their chosen scope(s), although contributions elsewhere are
+welcome, as well.
+
+### Contributors Group
+
+Contributors are listed in the README using the all-contributors bot.
+
+## GitHub Workflow
+
+For the day-to-day work on the BIDS specification, we currently abide by the
+following rules with the intention to:
+
+- Strive for consensus.
+- Promote open discussions.
+- Minimize the administrative burden.
+- Provide a path for when consensus cannot be made.
+- Grow the community.
+- Maximize the [bus factor](https://en.wikipedia.org/wiki/Bus_factor) of the
+  project.
+
+The rules outlined below are inspired by the
+[lazy consensus system used in the Apache Foundation](https://www.apache.org/foundation/voting.html)
+and heavily depends on
+[GitHub Pull Request Review system](https://help.github.com/articles/about-pull-requests/).
+
+## Rules
+
+1. Every modification of the specification (including a correction of a typo,
```

Comment:
Should this be "specification", or "project"/"software" since it's about BIDS-matlab and not the BIDS Specification? Or are you keeping the BIDS Specification Rules verbatim?

Reply:
That's why I had put this PR as draft and as a [Work in progress]. I want to go through all those changes to adapt them for us.

Looks like you did the job for me. :rocket: